README.md

```diff
@@ -66,31 +66,74 @@ JSONObject result = CoreApi.chargeTransaction(param);
 // SnapApi request with global config
 JSONObject result = SnapApi.createTransaction(param);
 ```
-
-All the request can accept an optional Config object. This is used if you want to set an others' config like idempotency key, proxy,
-or if you want to pass the server-key on each method, or use multiple account on each method.
+#### Per-request Configuration
+All the request can accept an optional Config object. This is used if you want to set an others' config like idempotency key, proxy, override/append notification url
+or want to use multiple account on each method. Please follow the steps given below.
 ```java
 import com.midtrans.Config;
 import com.midtrans.Midtrans;
 import com.midtrans.httpclient.CoreApi;
 import com.midtrans.httpclient.error.MidtransError;
 import org.json.JSONObject;
 
-Config configOptions = Config.builder()
+Config coreApiConfigOptions = Config.builder()
         .setServerKey("YOUR_SERVER_KEY")
         .setClientKey("YOUR_CLIENT_KEY")
         .setIrisIdempotencyKey("UNIQUE_ID")
         .setPaymentIdempotencyKey("UNIQUE_ID")
         .setProxyConfig(PROXY_CONFIG)
+        .setPaymentOverrideNotification("WEBHOOK_ENDPOINT")
         .setIsProduction(false)
         .build();
 
 // CoreApi request with config options
-JSONObject result = CoreApi.chargeTransaction(param, configOptions);
+JSONObject result = CoreApi.chargeTransaction(param, coreApiConfigOptions);
+
+Config snapConfigOptions = Config.builder()
+        .setServerKey("YOUR_SERVER_KEY")
+        .setClientKey("YOUR_CLIENT_KEY")
+        .setIrisIdempotencyKey("UNIQUE_ID")
+        .setPaymentIdempotencyKey("UNIQUE_ID")
+        .setProxyConfig(PROXY_CONFIG)
+        .setPaymentOverrideNotification("WEBHOOK_ENDPOINT")
+        .setIsProduction(false)
+        .build();
 
 // SnapApi request with config options
-JSONObject result = SnapApi.createTransaction(param, configOptions);
+JSONObject result = SnapApi.createTransaction(param, snapConfigOptions);
 ```
+
+In case you only have a use single account (credential-key) but you need to set the config options value dynamically from the config object. Please follow the steps given below.
```

Comment:
Fix grammar.
> In case you are using a single account API key but you need to set the config options value dynamically from the config object. Please follow the steps given below.